mxcube3/core/components/sampleview.py

```diff
@@ -560,10 +562,15 @@ def take_snapshots(self, snapshots=None, _do_take_snapshot=_do_take_snapshot):
                 hasattr(diffractometer_object, "set_phase")
                 and diffractometer_object.get_current_phase() != "Centring"
             ):
-                logging.getLogger("user_level_log").info(
-                    "Moving Diffractometer to CentringPhase"
+                use_custom_snapshot_routine = diffractometer_object.get_property(
```

Comment:
i wonder if we should keep this get_prop inside the dm object.

Also, would not this be HWR.beamline.diffractometer.....

edit: wait... is this actually in use?